internal/fs/fs_test.go

```diff
@@ -21,25 +22,35 @@ func TestHasFilepathPrefix(t *testing.T) {
 	}
 	defer os.RemoveAll(dir)
 
+	// dir2 is the same as dir but with different capitalization on Windows to
+	// test case insensitivity
```

Comment:
I added the comments you asked for (and some more). I do not know if you get notified of commits or I have to call them out in comments like this...